### PR TITLE
Feat/initialize frontend

### PR DESCRIPTION
**Pull request by dimamik**

### CODE REVIEW

**Comment by Akatroj** [comment]
LGTM

**Comment by Szustarol** [comment]
baza

**Review by Pirgo — APPROVED** [review]
LGTM

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,52 +1,62 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
-  const [count, setCount] = useState(0);
+  // const [roomId, setRoomId] = useState('');
 
-  const [roomId, setRoomId] = useState('');
+  // const socket = io('http://localhost:4000');
 
-  const socket = io('http://localhost:4000');
```
Comment (mario11-wiet) [inline client/src/App.tsx:11]: 3000 ?

**Comment by DawidGrapa** [comment]
Rocket Science (Frontend department)

**Review by Szustarol — APPROVED** [review]

**Review by DawidGrapa — APPROVED** [review]

File: client/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                <Row>
+                <Col><div className='footer'>Copyright &copy; ~~VaporWIET~~</div></Col>
+                </Row>
+            </Container>
+        </footer>
+    </div>
+  )
+}
+
+export default Footer
```
Comment (mario11-wiet) [inline client/src/components/Footer.tsx:18]: never used

**Review by Akatroj — APPROVED** [review]

**Review by mario11-wiet — APPROVED** [review]

File: client/src/components/Footer.tsx (hunk 2 above)
Comment (Akatroj) [reply client/src/components/Footer.tsx:18]: like your brain

**Pull request merged**